src/scripts/add_claim_subject_column.js

```diff
@@ -0,0 +1,38 @@
+const { createConnection, TableColumn } = require('typeorm');
+const { JWT } = require('@ew-did-registry/jwt');
+const { Keys } = require('@ew-did-registry/keys');
+const { Claim } = require('../../dist/modules/claim/claim.entity.js');
+
+(async function () {
+  const connection = await createConnection({
+    // had to copy from orgmconfig because typeorm doesn't detect postgres driver
+    type: "postgres",
+    host: "localhost",
+    port: 5432,
+    username: "postgres",
+    password: "password",
+    database: "dev",
+    "entities": [
+      "dist/**/*.entity.js"
+    ]
+  });
+  const claimsRepository = connection.getRepository(Claim);
+  let claims = await claimsRepository.find();
+
+  const queryRunner = connection.createQueryRunner();
+  if (!(await queryRunner.hasColumn("claim", "subject"))) {
+    await queryRunner.addColumn("claim", new TableColumn({ name: 'subject', type: 'varchar', default: "''" }));
```

Comment:
Why adding the column via this script instead of typeorm migration? IMO, my first impression is that a typeorm migration would be better so that schema changes are in one place.